[CUBRIDMAN-4] [en, ko] create security.rst file for the newly added CUBRID Security section.

hornetmj opened the pull request:
Create security.rst file for CUBRID Security section. Please let me know if anyone needs to register as a reviewer.

hornetmj committed:
[CUBRIDMAN-4] [en] create security.rst file for the newly added CUBRID Security section.

hornetmj committed:
[CUBRIDMAN-4] [ko] create security.rst file for the newly added CUBRID Security section.

wrlawodms commented on en/security.rst:
```diff
@@ -0,0 +1,24 @@
+
+:meta-keywords: security, cubrid security, authorization, acl, access control list, ssl, secure socket layer, packet encryption, tde, transparent data encryption, cubrid tde
+:meta-description: CUBRID Security includes authorization for specific objects, access control to a database, packet encryption, data encryption, etc. Users can manage user data more safely by using CUBRID Security features.
+
```
I think "CUBRID Security" has to be the first, root header. like:
```
*****************
CUBRID Security
*****************
```
and other section headers have to be modified like:
```
CUBRID Authorization
====================
```

mhoh3963 replied:
I hope to change order and name of section as following. what do you think ?
- Packet Encryption
- ACL (Access Control List)
- Authorization
- TDE (Transparent Data Encryption)

hornetmj replied:
@mhoh3963 I think it is also good for me.

hornetmj replied:
@mhoh3963 have applied it.

hornetmj replied:
@mhoh3963 @wrlawodms have applied it.

wrlawodms commented:
The links and decryption for "TDE Security" in other pages like "Introduction to Manual -> Manual Contents" are going to be added in another PR?

wrlawodms commented on en/security.rst:
```diff
@@ -0,0 +1,24 @@
+
+:meta-keywords: security, cubrid security, authorization, acl, access control list, ssl, secure socket layer, packet encryption, tde, transparent data encryption, cubrid tde
```
IMO, it would be better to add the following keywords: 
- data-at-rest encryption
- kernel encryption
- engine encryption
- key management

hornetmj replied:
Ok. I will apply them. And I think you can apply the necessary meta-keywords later in the your PR.

kisoo-han replied:
I want add 'tls' in the keywords like
"ssl, tls, secure socket layer, ..."

hornetmj replied:
@kisoo-han @wrlawodms have applied it.

mhoh3963 commented on ko/security.rst:
```diff
@@ -0,0 +1,24 @@
+:meta-keywords: security, cubrid security, authorization, acl, access control list, ssl, secure socket layer, packet encryption, tde, transparent data encryption, cubrid tde
+:meta-description: CUBRID Security includes authorization for specific objects, access control to a database, packet encryption, data encryption, etc. Users can manage user data more safely by using CUBRID Security features.
+
+********************
```
I hope to change order and name of section as following. what do you think ?
- 패킷암호화
- 서버접근제어
- 권한관리
- TDE (Transparent Data Encryption)

hornetmj replied:
@mhoh3963 I think it is also good for me.

hornetmj replied:
@mhoh3963 Do you want to translate 'CUBRID security' as 'CUBRID 보안'? Or do you have any other recommendations?

mhoh3963 replied:
@hornetmj  I like 'CUBRID 보안'.

hornetmj replied:
@mhoh3963 have applied it.

hornetmj replied:
@mhoh3963 have applied it.

hornetmj commented:
> The links and decryption for "TDE Security" in other pages like "Introduction to Manual -> Manual Contents" are going to be added in another PR?

@wrlawodms I think It should be included as CUBRID Security in the Manual Contents section. I referred to CUBRID HA, but this section is missing. I'll add it in this PR.

hornetmj committed:
apply feedback

1. add CUBRID Security section as root header.
2. use Korean in the ko path.
3. change the sequence and name of each header.

hornetmj committed:
apply feedback

- insert the requested meta-keywords.

hornetmj committed:
apply feedback

- add a link and brief explanation about CUBRID Security in the 'Manual Contents' of the 'Introduction to Manual' section.

hornetmj committed:
apply feedback

- add a description of CUBRID Security section.

charischau commented on en/toc.rst:
```diff
@@ -21,6 +21,8 @@ The contents of the CUBRID Database Management System (CUBRID DBMS) product manu
 
 *   :doc:`admin/index`: This chapter provides instructions on how to create, drop, back up, restore and migrate a database, configuring globalization, and executing CUBRID HA. Also it includes instructions on how to use the  **cubrid** utility, which starts and stops the server, broker, and CUBRID Manager server, etc. Also, this chapter provides instructions on setting system parameters that may  ...
```
1. Also it includes... --> Also, (comma added)

2. Also, this chapter provides instructions on setting system parameters that may influence the performance. --> 
In addition, this chapter provides..,  ( you have use 'also' in the last sentence, probably would be better to change 'also' to another word such as 'in addition,')

hornetmj replied:
@charischau Thanks for your feedback. But, this part has not been modified from this PR.

charischau replied:
okay sorry didnt know that, then I guess there are no problem with other part.
